### PR TITLE
npm: move dependencies to dev

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,3 +50,9 @@ deploy:
     on:
       branch: master
       python: '2.7'
+  - provider: script
```
Comment (michamos) [inline .travis.yml:53]: this is a duplicate I think

Comment (drjova) [reply .travis.yml:53]: I want to run this on ``tags`` for the npm package

Comment (drjova) [reply .travis.yml:53]: https://docs.travis-ci.com/user/deployment-v2/conditional#tag

Comment (harunurhan) [reply .travis.yml:53]: I don't understand why we would need this.

Comment (drjova) [reply .travis.yml:53]: to avoid having versions like ``61.3.0-2-g3e91dc0``

**Pull request merged**